ftest.sh

```diff
@@ -304,6 +304,18 @@ if ${SETUP_ONLY:-false}; then
     exit 0
 fi
 
+# check if slurm needs to be configured for soak
+
+if [[ $TEST_TAG_ARG == *soak* ]]; then
+    IFS="," read -r -a CONTROL_NODE <<< "$TEST_NODES"
```

Comment:
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]
(lint) The surrounding quotes actually unquote this. Remove or escape them. [SC2027]